.github/workflows/broadcast-frontend-hash_modify-file.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+FILE="$1"
+DFX_VERSION="$2"
+NEW_HASH="$3"
+
+awk -v dfx_version="$DFX_VERSION" -v new_hash="$NEW_HASH" '
+BEGIN { OFS="" }
+
+/const WHITELISTED_WASMS: \[&str; [0-9]+\] = \[/ {
+    split($0, arr, ";")
+    split(arr[2], subarr, "]")
+    num = subarr[1] + 1
+    print arr[1] "; " num "]" subarr[2]
```

Comment:
When I ran this script locally, the resulting diff had a trailing whitespace after the final `[`.  This might cause the PR to fail CI, if it's making a `cargo fmt` check.